packages/discovery-provider/src/queries/get_users_account.py

```diff
@@ -27,9 +27,7 @@ def _get_user_from_wallet(session, wallet: str):
     # Create initial query
     base_query = session.query(User)
     # Don't return the user if they have no wallet or handle (user creation did not finish properly on chain)
-    base_query = base_query.filter(
-        User.is_current == True, User.wallet != None, User.handle != None
-    )
+    base_query = base_query.filter(User.is_current == True, User.wallet != None)
```

Comment:
If I'm not mistaken, this query isn't used during sign-in on client anymore, as we hit the `/users/account` endpoint, the query for which is here: https://github.com/AudiusProject/audius-protocol/blob/7f49a62a38ae88866c9905abc583d45b754851c6/packages/discovery-provider/src/queries/get_users_account.py#L112

Reply:
Oops, I submitted this too quickly. However, I do think the outer function will throw if no wallet is provided?

Reply:
I submitted too quickly. Same file 🤦